docs/templates/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+
+<html lang="en">
+    <head>
+        <title>Redirecting to master branch</title>
+        <meta charset="utf-8" />
+        <meta
+            http-equiv="refresh"
+            content="0; url=./{{ latest.name }}/index.html"
+        />
+        <link rel="canonical" href="./{{ latest.name }}/index.html" />
+    </head>
```

Comment:
If I remember correctly `<meta http-equiv="refresh" content="0; url=..." />` is a non-standard extension, although supported by most browsers.
```suggestion
        <noscript><meta
            http-equiv="refresh"
            content="0; url=./{{ latest.name }}/index.html"
        /></noscript>
        <link rel="canonical" href="./{{ latest.name }}/index.html" />
    </head>
    <body onload="window.location.replace('./{{ latest.name }}/index.html')">
        If you are not redirected automatically, click <a href="./{{ latest.name }}/index.html">this link</a>.
    </body>
```